sandbox/docker_entrypoint.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+# Runs the command provided as arguments, but attempts to configure permissions first.
+
+important_read_dirs=("/grist" "/persist")
+important_write_dirs=("/persist")
+important_dirs=( "${important_read_dirs[@]}" "${important_write_dirs[@]}" )
+current_user_id=$(id -u)
+
+# We want to avoid running Grist as root if possible.
+# Try to setup permissions and de-elevate to a normal user.
+if [[ $current_user_id == 0 ]]; then
+  target_user=${GRIST_DOCKER_USER:-grist}
+  target_group=${GRIST_DOCKER_GROUP:-grist}
+
+  for dir in "${important_dirs[@]}"; do
+    # Make sure the target user owns everything that Grist needs read/write access to.
+    find "$dir" ! -user "$target_user" -exec chown "$target_user" "{}" +
```

Comment:
Why not using `chown -R "$target_user" "$dir"` instead? For performance reasons?

Reply:
Yeah, this line is derived from the postgres docker image:
https://github.com/docker-library/postgres/blame/29d4a29d8f750c7c55a22199311520de1e1f7f1e/docker-entrypoint.sh#L51

Which was added a time save / performance boost in this PR:
https://github.com/docker-library/postgres/pull/463